packages/opentelemetry-resources/src/platform/node/detectors/AwsEc2Detector.ts

```diff
@@ -44,13 +44,17 @@ class AwsEc2Detector implements Detector {
       const {
         accountId,
         instanceId,
+        instanceType,
         region,
+        availabilityZone,
       } = await this._awsMetadataAccessor();
       return new Resource({
         [CLOUD_RESOURCE.PROVIDER]: 'aws',
         [CLOUD_RESOURCE.ACCOUNT_ID]: accountId,
         [CLOUD_RESOURCE.REGION]: region,
+        [CLOUD_RESOURCE.ZONE]: availabilityZone,
```

Comment:
It would be nice if you update the test: https://github.com/justinwalz/opentelemetry-js/blob/feature/update-aws-detector/packages/opentelemetry-resources/test/detectors/AwsEc2Detector.test.ts#L57

Reply:
Sure thing.